### PR TITLE
Fix the link and content for upgrade instruction

### PR DESCRIPTION
**Pull request by reasonerjt**
Signed-off-by: Daniel Jiang <jiangd@vmware.com>

Thank you for contributing to Velero!

# Please add a summary of your change

# Does your change fix a particular issue?

Fixes #(issue)

# Please indicate you've done the following:

- [X] [Accepted the DCO](https://velero.io/docs/v1.5/code-standards/#dco-sign-off). Commits without the DCO will delay acceptance.
- [X] [Created a changelog file](https://velero.io/docs/v1.5/code-standards/#adding-a-changelog) or added `/kind changelog-not-required`.
- [X] Updated the corresponding documentation in `site/content/docs/main`.

### CODE REVIEW

**Review by sseago — APPROVED** [review]

**Review by zubron — APPROVED** [review]
Thanks for fixing this!

**Review by jenting — APPROVED** [review]

**Comment by zubron** [comment]
I'm going to hold off on merging this for now. If we are going to create another RC then we can include it, otherwise it will be safe to merge this once we release 1.7 as the website always uses the `main` branch. The documentation changes don't need to exist on the release branch.

**Comment by reasonerjt** [comment]
Per discussion, we are gonna merge this one to make content consistent

**Pull request merged**
